### PR TITLE
Raw output prefix

### PR DESCRIPTION
**Pull request by wild-endeavor**
# TL;DR
This will pipe through the raw output data prefix, as set by the [flytekit PR](https://github.com/lyft/flytekit/pull/146/files#diff-937f5bf70e8c655b5e2731a288bc3db8R17), through admin/propeller/propeller's node execution context, to container based plugins that call the replaceTemplateArgs.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
* Add regex, use regex to fill in the container args.
* Also accidentally ran `make generate` so some stuff got updated.

## Tracking Issue
https://github.com/lyft/flyte/issues/211

## Follow-up issue
NA

### CODE REVIEW

File: go/tasks/pluginmachinery/utils/template_test.go [hunk 1]
```diff
@@ -338,4 +342,18 @@ func TestReplaceTemplateCommandArgs(t *testing.T) {
 			"output/blah",
 		}, actual)
 	})
+
+	t.Run("sub raw output data prefix", func(t *testing.T) {
```
Comment (kumare3) [inline go/tasks/pluginmachinery/utils/template_test.go:346]: can we create a test, where we are replacing other things with raw-output prefix?

Comment (wild-endeavor) [reply go/tasks/pluginmachinery/utils/template_test.go:346]: mmmm... yeah.  sorry does not compute.  can you elaborate?

Comment (wild-endeavor) [reply go/tasks/pluginmachinery/utils/template_test.go:356]: done
